indexer/crawler.go

```diff
@@ -214,12 +215,19 @@ func (c *CrawlDispatcher) AddToCatchupQueue(ctx context.Context, host *models.PD
 		panic("not okay")
```

Comment:
Why do we panic here?

Reply:
I should make those messages better... but we need to be aware of which PDS the work is coming from for scheduling purposes